pinot-tools/src/main/resources/conf/pinot-broker.conf

```diff
@@ -0,0 +1,33 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+# Pinot Cluster name
+pinot.cluster.name=pinot-quickstart
```

Comment:
Should we have a single conf file for the entire cluster, so that user doesn't need to maintain three/four separate conf files?

Reply:
make sense, I will add one for StartServiceManager

Reply:
we don't have a single conf file right now as the StartServiceManager requires a role to start pinot, which is the same key across all the configs.

Current way to start all-in-one container is to give 3 conf files, e.g.
```
bin/pinot-admin.sh StartServiceManager -bootstrapConfigPaths conf/pinot-controller.conf conf/pinot-broker.conf conf/pinot-server.conf
```